code/modules/hivemind/wires.dm

```diff
@@ -389,6 +389,11 @@
 			if(is_neotheology_disciple(L))
 				return
 
+			to_world("Wires begin to wreathe around [L], starting the process of converting them into part of the hivemind.") //We tell people to get them off the wires
+			sleep(90 SECONDS) //We give them 90 seconds to be pulled off the wires
```

Comment:
Sleep is really bad to use, as it stops here for 90 seconds and returns the proc constantly asking `is 90nseconds past?`

Reply:
What's a better alternative to use?